src/com/tambolaassingment/Player.java

```diff
@@ -21,48 +23,48 @@ public Player(String name, TicketChecker ticketChecker) {
         }
 
     }
-    public  HashMap<Integer,Boolean> getTicket(){
-        return  ticket;
-    }
-    boolean housie(HashMap<Integer,Boolean> ticket){
-            for ( Boolean key : ticket.values() ) {
-                if(!(boolean) key) {
-                    return  false;}
-            }
-            return true;
+
+    public Map<Integer, Boolean> getTicket() {
```

Comment:
Please remove this method as I don't think that this method is anywhere.

Reply:
done